src/virtual-machine.js

```diff
@@ -1005,12 +1005,16 @@ class VirtualMachine extends EventEmitter {
         );
 
         const variables = Object.keys(variableMap).map(k => variableMap[k]);
+        const wkspcComments = Object.keys(this.editingTarget.comments)
```

Comment:
Can I buy a vowel?

;) I think we should stick with the full word as much as possible